packages/atlas/src/joystream-lib/extrinsics.ts

```diff
@@ -465,7 +469,105 @@ export class JoystreamLibExtrinsics {
       block,
     }
   }
+  /*
+    === Channel payouts ===
+  */
+
+  getClaimableReward = async (
+    channelId: string,
+    cumulativeRewardEarned: StringifiedNumber | null,
+    nodeEndpoint: string,
+    payloadDataObjectId: string
+  ) => {
+    const payoutProof = await channelPayoutProof(
+      'URL',
+      `${nodeEndpoint}/api/v1/files/${payloadDataObjectId}`,
+      Number(channelId)
+    )
+
+    const cashout = new BN(payoutProof.cumulativeRewardEarned).sub(new BN(cumulativeRewardEarned || 0))
+    return cashout.toString()
+  }
+
+  claimRewardTx = async (
+    channelId: string,
+    memberId: MemberId,
+    cumulativeRewardEarned: StringifiedNumber | null,
+    nodeEndpoint: string,
+    payloadDataObjectId: string,
+    commitment: string
+  ) => {
+    await this.ensureApi()
 
+    const payoutProof = await channelPayoutProof(
+      'URL',
+      `${nodeEndpoint}/api/v1/files/${payloadDataObjectId}`,
+      Number(channelId)
+    )
+
+    const maxCashoutAllowed = await this.api.query.content.maxCashoutAllowed()
+    const minCashoutAllowed = await this.api.query.content.minCashoutAllowed()
+    const cashout = new BN(payoutProof.cumulativeRewardEarned).sub(new BN(cumulativeRewardEarned || 0))
+    const isPayoutProofVerified = verifyChannelPayoutProof(payoutProof) === commitment
+
+    if (maxCashoutAllowed.lt(cashout)) {
+      throw new JoystreamLibError({
+        name: 'FailedError',
+        message: `Channel cashout amount is too low to be claimed`,
```

Comment:
I think in this case it's `too high` actually? Maybe we could switch the condition around and make it `cashout.gt(maxCashoutAllowed)`? That's a bit easier to read IMO